presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -105,6 +105,42 @@ General Aggregate Functions
 
     Returns ``n`` smallest values of all input values of ``x``.
 
+.. function:: reduce_agg(inputValue T, initialState S, inputFunction(S, T, S), combineFunction(S, S, S)) -> S
+
+    Returns a single value reduced from input values. ``inputFunction`` and ``combineFunction`` will be invoked.
+    ``inputFunction`` takes the current state (initially ``initialState``) and the input value, and returns the new state.
+    ``combineFunction`` takes two states and returns the new state.
+    The value of final state is returned::
+
+        SELECT id, reduce_agg(value, 0, (a, b) -> a + b, (a, b) -> a + b)
+        FROM (
+            VALUES
+                (1, 3),
```

Comment:
Let's change these examples to have three values for each group, so that it's more clear how the aggregation works.